docs/services/navigation/_index.md

```diff
@@ -554,3 +554,128 @@ obstacles = await my_nav.get_obstacles()
 
 {{% /tab %}}
 {{< /tabs >}}
+
+## Concepts
+
+The following concepts are important to understand when utilizing the navigation service.
+Each concept is a type of relative or absolute measurement, taken by a [movement sensor](/components/movement-sensor/), which can then be utilized by your robot to navigate across space.
+
+Here's how to make use of the following types of measurements:
+
+- [Compass Heading](/services/navigation/#compass-heading)
+- [Orientation](/services/navigation/#orientation)
+- [Angular Velocity](/services/navigation/#angular-velocity)
+- [Position](/services/navigation/#position)
+- [Linear Acceleration](/services/navigation/#linear-acceleration)
+- [Linear Velocity](/services/navigation/#linear-velocity)
+
+### Compass Heading
+
+The following {{< glossary_tooltip term_id="model" text="models" >}} of [movement sensor](/components/movement-sensor/) take compass heading measurements:
+
+- [gps-nmea](/components/movement-sensor/gps/gps-nmea/)
+- [gps-nmea-rtk-pmtk](/components/movement-sensor/gps/gps-nmea-rtk-pmtk/)
+- [gps-nmea-rtk-serial](/components/movement-sensor/gps/gps-nmea-rtk-serial/)
+
+An example of a `Compass Heading` reading:
+
+``` go
+gps.GetCompassHeading{175}
+```
+
+The movement sensor API's [`GetCompassHeading()`](/components/movement-sensor/#getcompassheading) takes compass heading readings.
+If you want to read compass headings, [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
+Use compass heading readings to determine the *bearing* of your robot, or, the [cardinal direction](https://en.wikipedia.org/wiki/Cardinal_direction) that your robot is facing.
+
+### Orientation
+
+The following {{< glossary_tooltip term_id="model" text="models" >}} of [movement sensor](/components/movement-sensor/) take orientation measurements:
+
+- [imu-wit](/components/movement-sensor/imu/imu-wit/)
+- [imu-vectornav](/components/movement-sensor/imu/imu-vectornav/)
+
+An example of an `Orientation` reading:
+
+``` golang
+orientation, err := imuwit.Orientation(context.Background, nil)
+```
+
+The movement sensor API's [`GetOrientation()`](/components/movement-sensor/#getorientation) takes orientation readings.
+
+Use orientation readings to determine the orientation of an object in 3D space as an "orientation vector", or, its position within the [cartesian coordinate system](https://en.wikipedia.org/wiki/Cartesian_coordinate_system) relative to some specific `origin` point that you, the user, need to choose and configure for your robot in Viam's [frame system](/services/frame-system/).
```

Comment:
@randhid re. @JessamyT feedback-- if you get another chance to review, how can I phrase this accurately here (line 605)? What i mean to convey is that you configure a (0,0,0),0 point of origin for the world frame and then nest everything off of that

Reply:
".... , or, how it is rotated relative to an `origin` coordinate system around the x, y, and z axes. You can choose the `origin` reference frame by configuring it using Viam's frame system. The readings of the Orientation from the movement sensor will report orientations relative to that initial frame."

Optional: "If you do not configure a frame, the default is an at 0,0,0 with a rotation of 0 about the z axis. "
But then i think you'd have to explain that it updated with the frame system in motion and blah blah blah so I'd just link to other pages if you don't want to do that here.

Reply:
Making a suggestion that attempts to incorporate this feedback

Reply:
@JessamyT applied! Thank you for making suggestion!